database/create_prs.sql

```diff
@@ -1,68 +1,70 @@
-USE master 
-GO
-
-DROP DATABASE IF EXISTS PRS,
GO;
-
-CREATE DATABASE PRS
-
-USE PRS 
-GO
-
-CREATE TABLE Users
-(
-Id			INT				NOT NULL	IDENTITY PRIMARY KEY, 
-Username	VARCHAR(30)	    NOT NULL,	
-Password	VARCHAR(30)		NOT NULL, 
-Firstname	VARCHAR(30)		NOT NULL,
-Lastname	VARCHAR(30)		NOT NULL,
-Phone		VARCHAR(12)		NULL,		
-Email		VARCHAR(255)	NULL,
-IsReviewer	BIT				NOT NULL,
-IsAdmin		BIT				NOT NULL 
-);
-
-CREATE TABLE Vendors
-(
-Id			INT				NOT NULL	IDENTITY PRIMARY KEY,
-Code		VARCHAR(30)		NOT NULL,	
-Name		VARCHAR(30)		NOT NULL,
-Address		VARCHAR(30)		NOT NULL,
-City		VARCHAR(30)		NOT NULL,
-State		VARCHAR(2)		NOT NULL,
-Zip			VARCHAR(5)		NOT NULL,
-Phone		VARCHAR(12)		NULL,
-Email		VARCHAR(255)	NULL,
-);
-
-CREATE TABLE Products
-(
-Id			INT				NOT NULL	IDENTITY PRIMARY KEY, 
-PartNbr		VARCHAR(30)		NOT NULL,
-Name		VARCHAR(30)		NOT NULL,
-Price		DECIMAL(11,2)	NOT NULL,
-Unit		VARCHAR(30)		NOT NULL,
-PhotoPath	VARCHAR(255)	NULL,
-VendorId	INT				NOT NULL,
-);
-
-CREATE TABLE Requests
-(
-Id				INT					NOT NULL	IDENTITY PRIMARY KEY,
-Description		VARCHAR(80)			NOT NULL,
-Justification	VARCHAR(80)			NOT NULL,
-RejectionReason	VARCHAR(80)			NULL,
-DeliveryMode	VARCHAR(20)			NOT NULL	DEFAULT 'Pickup',
-SubmittedDate	DATE				NOT NULL	DEFAULT (GETDATE()),
-DateNeeded		DATE				NOT NULL,
-Status			VARCHAR(10)			NOT NULL	DEFAULT 'NEW',
-Total			DECIMAL(11,2)		NOT NULL	DEFAULT 0,
-UserId			INT					NOT NULL,
-);
-
-CREATE TABLE RequestLine
-(
-Id			INT			NOT NULL		IDENTITY PRIMARY KEY,
-RequestId	INT			NOT NULL,
-ProductId	INT			NOT NULL,
-Quantity	INT			NOT NULL		DEFAULT 1,
-);
\ No newline at end of file
+USE master 
+GO
+
+DROP DATABASE IF EXISTS PRS,
+GO;
```

Comment:
Remove the comma from after `prs` and replace it with a semicolon. Remove semicolon from `go`

Reply:
Hi Kait. I made the changes you requested. Please let me know what else I need to fix (I'm sure there is more lol). :)